### PR TITLE
CapricornConsultingGmbH.Vigoon version (1.0.0)

### PR DESCRIPTION
**Pull request by cctimoerdbruegger**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40665)

### CODE REVIEW

File: manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CapricornConsultingGmbH.Vigoon
+PackageVersion: (2.4.2559.0)
```
Comment (ItzLevvie) [inline manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: CapricornConsultingGmbH.Vigoon
+PackageVersion: (2.4.2559.0)
```
Comment (ItzLevvie) [inline manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CapricornConsultingGmbH.Vigoon
+PackageVersion: (2.4.2559.0)
```
Comment (ItzLevvie) [inline manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

Comment (ItzLevvie) [reply manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.installer.yaml:5]: This must match Control Panel.

File: manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml:5]: This must match Control Panel.

File: manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.yaml:5]: This must match Control Panel.

File: manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CapricornConsultingGmbH.Vigoon
+PackageVersion: (2.4.2559.0)
+PackageLocale: en-US
+Publisher: Capricorn Consulting GmbH
+PackageName: Vigoon
+License: Copyright (c) Capricorn Consulting GmbH. All rights reserved.
+ShortDescription: Vigoon App Platform - Version 2.4.2559.0
```
Comment (ItzLevvie) [inline manifests/c/CapricornConsultingGmbH/Vigoon/(2.4.2559.0)/CapricornConsultingGmbH.Vigoon.locale.en-US.yaml:10]:
```
ShortDescription: Vigoon App Platform - Version 2.4.2559.0
```
Please write a better description for what this application is about.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Make sure to update the version folder as well: https://github.com/microsoft/winget-pkgs/pull/40665#pullrequestreview-852795092; otherwise the application upgrade will end up in a upgrade loop.

You'll also need to update the description for the application.

**Comment by cctimoerdbruegger** [comment]
> Make sure to update the version folder as well: [#40665 (review)](https://github.com/microsoft/winget-pkgs/pull/40665#pullrequestreview-852795092); otherwise the application upgrade will end up in a upgrade loop.
> 
> You'll also need to update the description for the application.

How should i update the version folder? The package identify is not recognized by wingetcreate update comment. So i am not allowed to push an updated manifest. I can only make a new manifest pull request then?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**